index.js

```diff
@@ -28,7 +28,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let movie = [];
+  if (!movies.length) {
```

Comment:
This works great because we know movies is going to be an array but how can we write this to ensure against some bad data like falsy values as well?